### ISSUE TITLE
Make accessible all buttons

### ISSUE THREAD

**Issue by Eloi-Perez** [chingu-voyages/v42-bears-team-27#193]
**__Task Description__**
Buttons with Icons and no text content need to have the property `aria-label=""`

**Comment by Eloi-Perez** [comment]
I just found another aria-label missing, will commit to this branch shortly
